react/src/components/ContainerRegistryEditorModal.tsx

```diff
@@ -150,37 +90,21 @@ const ContainerRegistryEditorModal: React.FC<
                 ? null // unset
                 : values.password
               : undefined, // no change
+          extra:
+            _.isEmpty(values.extra) || _.isNil(values.extra)
```

Comment:
Is `isNil` a subset of `isEmpty`?

Reply:
`isNil` contains only validate for null or undefined. `isEmpty` was added to validate values such as the empty string ''.

Reply:
When `isNil(something)` is true, isEmpty(something` is always true. please remove `isNil` part.